### PR TITLE
Bug Fixes

### PR DESCRIPTION
**Pull request by Noztik**

### CODE REVIEW

File: lua/cinema/map/sv_mapprops.lua [hunk 1]
```diff
@@ -212,3 +212,13 @@ if MAPPROPS_RERUN then
 end
 
 MAPPROPS_RERUN = true
+
+hook.Add("OnEntityCreated","TripminePositionValidator",function(ent,dmginfo) --move this somewhere else
+	if ent:GetClass() == "npc_tripmine" then
+		timer.Simple(0,function()
+			if not util.IsInWorld(ent:GetPos()) then
```
Comment (swampservers) [inline lua/cinema/map/sv_mapprops.lua:219]: use `ent:TimerSimple(0, function() ...` to ensure the ent is still valid

File: lua/cinema/map/sv_mapprops.lua [hunk 2]
```diff
@@ -212,3 +212,13 @@ if MAPPROPS_RERUN then
 end
 
 MAPPROPS_RERUN = true
+
+hook.Add("OnEntityCreated","TripminePositionValidator",function(ent,dmginfo) --move this somewhere else
```
Comment (swampservers) [inline lua/cinema/map/sv_mapprops.lua]: the dmginfo param is not supposed to be there

File: lua/cinema/weapons/weapon_spraypaint/shared.lua [hunk 3]
```diff
@@ -67,6 +67,7 @@ end
 
 function SWEP:PrimaryAttack()
     local ply = self:GetOwner()
+    if not (IsValid(ply) and ply:Alive()) then return end
```
Comment (swampservers) [inline lua/cinema/weapons/weapon_spraypaint/shared.lua:70]: is this to fix an error on server or client? it seems like the weapon shouldn't be able to primary attack unless it has a living owner, unless maybe the localplayer is invalid on client somehow

Comment (Noztik) [reply lua/cinema/weapons/weapon_spraypaint/shared.lua:70]: an error is thrown when a person dies while using the spraypaint

Comment (swampservers) [reply lua/cinema/weapons/weapon_spraypaint/shared.lua:70]: Oh I see why. The check should be moved down to line 117-ish i think

**Pull request merged**